src/external/adobe_coeff.c

```diff
@@ -183,6 +183,7 @@ static void dt_dcraw_adobe_coeff(const char *name, float cam_xyz[1][12])
     { "Fujifilm GFX 50R", { 11756,-4754,-874,-3056,11045,2305,-381,1457,6006 } },
     { "Fujifilm GFX 50S", { 11756,-4754,-874,-3056,11045,2305,-381,1457,6006 } },
     { "Fujifilm GFX 100", { 16212,-8423,-1583,-4336,12583,1937,-195,726,6199 } },
+    { "Fujifilm GFX100S", { 16212,-8423,-1583,-4336,12583,1937,-195,726,6199 } },
```

Comment:
Have you perhaps confirmed these w/ the latest Adobe DNG Converter as well?

Reply:
> Have you perhaps confirmed these w/ the latest Adobe DNG Converter as well?

Sorry I don't have any Windows or MacOS computer. So no I didn't

Reply:
No probs, I just used one from RPU on my Windows machine, and it check out.